<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ENSDFDownloader</class>
 <widget class="QDialog" name="ENSDFDownloader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>266</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Database Download</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Kaihen uses the &lt;a href=&quot;http://www.nndc.bnl.gov/ensdf/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;ENSDF database&lt;/span&gt;&lt;/a&gt; as source for nuclear decay information. Please press the download button to automatically retrieve the necessary files.&lt;/p&gt;&lt;p&gt;If you already have a copy of the ENSDF files and do not want to download it again you can also select the appropriate local folder.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="downloadButton">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>Automatically Download Database</string>
     </property>
     <property name="icon">
      <iconset resource="kaihen.qrc">
       <normaloff>:/download.png</normaloff>:/download.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="buttonPage">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="localButton">
           <property name="text">
            <string>Select Local Copy</string>
           </property>
           <property name="icon">
            <iconset resource="kaihen.qrc">
             <normaloff>:/document-open-folder.png</normaloff>:/document-open-folder.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="setupButton">
           <property name="text">
            <string>Adjust Download Settings</string>
           </property>
           <property name="icon">
            <iconset resource="kaihen.qrc">
             <normaloff>:/configure.png</normaloff>:/configure.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="closeButton">
           <property name="text">
            <string>Close</string>
           </property>
           <property name="icon">
            <iconset resource="kaihen.qrc">
             <normaloff>:/dialog-close.png</normaloff>:/dialog-close.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="progressPage">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>8</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="progressLabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="ftpCancelButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Cancel</string>
           </property>
           <property name="icon">
            <iconset resource="kaihen.qrc">
             <normaloff>:/dialog-cancel.png</normaloff>:/dialog-cancel.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="kaihen.qrc"/>
 </resources>
 <connections/>
</ui>
